tutorials/vr/vr_starter_tutorial/vr_starter_tutorial_part_one.rst

```diff
@@ -0,0 +1,1146 @@
+- You can make rough models using a tool like `Google Blocks <https://vr.google.com/blocks/>`_, and then refine in another 3D modelling program.
+- Test often, as the assets can look dramatically different in VR than on a flat screen!
+
+Throughout the course of this tutorial, we will cover:
+
+- How to tell Godot to run in VR.
+- How to make a teleportation locomotion system that uses the VR controllers.
+- How to make a artificial movement locomotion system that uses the VR controllers.
+- How to create a :ref:`RigidBody <class_RigidBody>`-based system that allows for picking up, dropping, and throwing RigidBody nodes using the VR controllers.
+- How to create simple destroyable target.
+- How to create some special :ref:`RigidBody <class_RigidBody>`-based objects that can destroy the targets.
+
+.. tip:: While this tutorial can be completed by beginners, it is highly
+          advised to complete :ref:`doc_your_first_game`,
+          if you are new to Godot and/or game development.
+          
+          **Some experience with making 3D games is required** before going through this tutorial series.
+          This tutorial assumes you have experience with the Godot editor, GDScript, and basic 3D game development.
+          A OpenVR-ready headset and two OpenVR-ready controllers are required.
+          
+          This tutorial was written and tested using a Windows Mixed Reality headset and controllers. This project has also been tested on the HTC Vive. Code adjustments may be required
+          for other VR Headsets, such as the Oculus Rift.
+
+The Godot project for this tutorial is found on the `OpenVR GitHub repository <https://github.com/GodotVR/godot_openvr_fps>`_. The starter assets for this tutorial can be found in the releases
+section on the GitHub repository. The starter assets contain some 3D models, sounds, scripts, and scenes that are configured for this tutorial.
+
+.. note:: **Credits for the assets provided**:
+          
+          - The sky panorama was created by `CGTuts <https://cgi.tutsplus.com/articles/freebie-8-awesome-ocean-hdris--cg-5684>`_.
+          
+          - The font used is Titillium-Regular 
+          - - The font is licensed under the SIL Open Font License, Version 1.1
+          
+          - The audio used are from several different sources, all downloaded from the Sonniss #GameAudioGDC Bundle (`License PDF <https://sonniss.com/gdc-bundle-license/>`_) 
+          - - The folders where the audio files are stored have the same name as folders in the Sonniss audio bundle.
+          
+          - The OpenVR addon was created by `Bastiaan Olij <https://github.com/BastiaanOlij>`_ and is released under the MIT license. It can be found both on the `Godot Asset Library <https://godotengine.org/asset-library/asset/150>`_ and on `GitHub <https://github.com/GodotVR/godot-openvr-asset>`_.
+          
```

Comment:
While my code is MIT, the library Valve supplies isn't. We may need to make a note of that.